lib/src/cpp/domain_participant/DomainParticipant.cpp

```diff
@@ -259,7 +264,91 @@ void set_name(
         const std::string& profile_id,
         const std::string& name)
 {
-    throw Unsupported("Unsupported");
+    // Xerces document manage XML elements
+    xercesc::DOMDocument* doc = nullptr ;
+
+    // XML nodes and values
+    xercesc::DOMNode* profiles_node = nullptr ;
+    xercesc::DOMNode* participant_node = nullptr ;
+    xercesc::DOMNode* rtps_node = nullptr ;
+    xercesc::DOMNode* name_node = nullptr ;
```

Comment:
I understand that this nodes are going to be specific for each element. Should we have the variables in the infrastructure class and use only those that we need? Should we abstract them in some way? Should we keep them in stack?